changlog.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+cd 'src'
+
+# Define the new version number (you can pass this as an argument or set it dynamically)
+# Get the last created tag from GitHub
+new_version=$(curl -s "https://api.github.com/repos/sesamyab/wordpress-sesamy/tags" | jq -r '.[7].name')
```

Comment:
I'm not a sh expert but wouldn't `.[7].name` pick the name property of the 8th element of the array returned by the GitHub tags endpoint? Why the 8th? Is this one always the latest?

Reply:
Yes, as previously discussed, you may need to discuss this with Markus first before proceeding.

Reply:
@markusahlstrand is it okay to remove the tags starting with the prefix `v`? We're not using them for any particular reason, which could prevent future issues for checking a specific index in the tags instead of the latest.

Reply:
Yes, for me it's not a problem :)